package-system/AWSNativeSDK/FindAWSNativeSDK.cmake.Mac

```diff
@@ -88,59 +87,77 @@ function(ly_declare_aws_library)
 endfunction()
 
 
-#### Common ####
+#### CRT ####
 if(LY_MONOLITHIC_GAME)
     ly_declare_aws_library(
         NAME 
-            Common
-        LIB_FILE 
-            aws-c-common
-    )       
-else()
-    ly_declare_aws_library(
-        NAME 
-            Common
+            AWSCrt
         LIB_FILE 
-            aws-c-common
+            aws-crt-cpp
         BUILD_DEPENDENCIES
-            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-common${CMAKE_SHARED_LIBRARY_SUFFIX}
-            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-common.0unstable${CMAKE_SHARED_LIBRARY_SUFFIX}
-            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-common.1.0.0${CMAKE_SHARED_LIBRARY_SUFFIX}
-        RUNTIME_DEPENDENCIES
-            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-common.0unstable${CMAKE_SHARED_LIBRARY_SUFFIX}
-            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-common.1.0.0${CMAKE_SHARED_LIBRARY_SUFFIX}
-    )       
-endif()
-
-#### Checksums ####
-ly_declare_aws_library(
-    NAME 
-        Checksums
-    LIB_FILE 
-        aws-checksums
-)
-
-#### EventStream ####
-if(LY_MONOLITHIC_GAME)
-    ly_declare_aws_library(
-        NAME 
-            EventStream
-        LIB_FILE 
-            aws-c-event-stream
-        BUILD_DEPENDENCIES 
-            3rdParty::AWSNativeSDK::Checksums
+            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-s3.a
+            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-auth.a
+            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-http.a
+            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-io.a
+            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-mqtt.a
+            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-event-stream.a
+            ${AWSNATIVE_SDK_LIB_PATH}/libaws-checksums.a
+            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-common.a
+            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-compression.a
+            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-cal.a
     )
 else()
     ly_declare_aws_library(
         NAME 
-            EventStream
+            AWSCrt
         LIB_FILE 
-            aws-c-event-stream
+            aws-crt-cpp
         BUILD_DEPENDENCIES
-            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-event-stream${CMAKE_SHARED_LIBRARY_SUFFIX}
-            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-event-stream.0unstable${CMAKE_SHARED_LIBRARY_SUFFIX}
-            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-event-stream.1.0.0${CMAKE_SHARED_LIBRARY_SUFFIX}
-            3rdParty::AWSNativeSDK::Checksums
+            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-s3.0unstable.dylib
```

Comment:
Nit: Is it possible to use ${CMAKE_SHARED_LIBRARY_SUFFIX} instead of the hardcoded .dylib here?

Reply:
no, we should not, those variables are used if there is only one FindAWSNativeSDK.cmake.template, so suffix can be replaced properly for different platforms.
But we have different FindAWSNativeSDK for each platform, and they are not template anymore. So we should hardcode, like mac uses .dylib and .a, windows uses .dll and .lib, etc.